dockerfiles/jena-fuseki2-docker/README.md

```diff
@@ -0,0 +1 @@
+Commit: 26153afe9a229bb7a609d5406d899eb240ab385e
```

Comment:
This is more than a little bit cryptic. Maybe this README could explain that these files were copied from the Jena repo?

Reply:
+1!

I thought I had written about it somewhere. I will have a look and add it to this README.

Reply:
This still needs to be done, right?